src/main/java/com/uid2/admin/vertx/service/OperatorKeyService.java

```diff
@@ -184,6 +187,11 @@ private void handleOperatorAdd(RoutingContext rc) {
                 return;
             }
 
+            Set<Role> roles = RequestUtil.getRoles(rc.queryParam("roles").get(0)) == null ? Set.of() : RequestUtil.getRoles(rc.queryParam("roles").get(0));
+            if (roles.stream().allMatch(Objects::isNull)) {
+                ResponseUtil.error(rc, 400, "No roles specified");
```

Comment:
so when a user wants to add a new Operator, the OPERATOR role must be explicitly specified in the textbox or otherwise it will hit this error here? Is that what we want to do (seems a bit cumbersome)? 

If we keep it this way, the error msg should say something like "No roles specified - at least OPERATOR role must be specified here" to clarify what the users need to input

Reply:
hmmm, i guess with add new operator we should be able to leave the role to be empty, I will remove the check here